src/lib/final_report.ml

```diff
@@ -417,6 +443,11 @@ module To_workflow
             let wf = get_vcf repr in
             let title = sprintf "VCF: %s" name in
             title, Some wf#product#path)
+        @ (Option.value_map ~default:(["Kallisto", None]) kallisto ~f:(fun ks ->
```

Comment:
Sorry for the delay; you are correct: just the empty list for no results is what you want here!